src/Components/PaperEditor/ProgrammeScriptContainer/index.js

```diff
@@ -148,6 +151,7 @@ const ProgrammeScriptContainer = (props) => {
     }
   }, [ elements, paperEdits ]);
 
+  //when paperEdits are updated, update transcripts
```

Comment:
Thank you for adding these comments, I think they make it much quicker to understand what's happening with the code 💯